src/createServer.js

```diff
@@ -1,3 +1,62 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const notification = require('./notifications')
+const { convertToCase } = require('./convertToCase');
+
+const supportedCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    const { pathname, searchParams } = new URL(req.url, `http://${req.headers.host}`);
+    const toCase = searchParams.get('toCase');
+    const textToConvert = pathname.slice(1);
```

Comment:
You can move it to the separate function & file
```suggestion
  const { ... } = ...( ... );
```

Reply:
Done